executor/update.go

```diff
@@ -100,10 +116,26 @@ func (e *UpdateExec) exec(ctx context.Context, schema *expression.Schema, row, n
 			continue
 		}
 
+		var updatedData []types.Datum
+		v, ok = e.updatedRowData[content.TblID].Get(handle)
+		if !ok {
+			updatedData = make([]types.Datum, len(oldData))
+			copy(updatedData, oldData)
```

Comment:
can we avoid this alloc + copy when it's a single table update without swap any value(e.g. update t set v = 2 where k = 1)